src/test/java/jenkins/branch/BranchCategoryFilterTest.java

```diff
@@ -147,21 +147,21 @@ public void filter_specificCategory() throws Exception {
 
     }
 
-    private static abstract class TopLevelJob extends Job<TopLevelJob, TopLevelRun> implements TopLevelItem {
+    public static abstract class TopLevelJob extends Job<TopLevelJob, TopLevelRun> implements TopLevelItem {
```

Comment:
If this is purely for mockito to not blow up on these, then internal should be okay. 

```suggestion
    static abstract class TopLevelJob extends Job<TopLevelJob, TopLevelRun> implements TopLevelItem {
```